pom.xml

```diff
@@ -38,9 +56,19 @@
     <maven.compiler.target>17</maven.compiler.target>
     <lombok.version>1.18.30</lombok.version>
     <eip.client.version>2.1.0-SNAPSHOT</eip.client.version>
-    <spotless.version>2.30.0</spotless.version>
-    <palantirJavaFormat.version>1.1.0</palantirJavaFormat.version>
     <openmrs.watcher.version>4.1.0-SNAPSHOT</openmrs.watcher.version>
+    <camel.openmrs.fhir.version>4.1.0-SNAPSHOT</camel.openmrs.fhir.version>
+    <fhir-r4.version>7.0.0</fhir-r4.version>
+    <camel.version>4.6.0</camel.version>
+    <test.containers.version>1.19.3</test.containers.version>
+    <ozone.version>1.0.0-SNAPSHOT</ozone.version>
+    <skipTests>false</skipTests>
```

Comment:
This is redundant

Reply:
Including these other skip*Test properties

Reply:
These configs can we useful if someday we decide to mark these as `true`